addons/web/static/src/model/relational_model/static_list.js

```diff
@@ -148,34 +148,6 @@ export class StaticList extends DataPoint {
         return this.handleField && this.orderBy.length && this.orderBy[0].name === this.handleField;
     }
 
-    /**
-     * TODO: We should probably delete this function.
```

Comment:
@aab-odoo do you prefer if the method is removed with its only use or should I only remove it in master ?

Also, is it expected that CLEAR magic commands are not supported by the javascript framework ?
I don't really require the change I did to support it since I can use a SET command with an empty list, but I wanted to highlight that difference between server and client-side commands.

Reply:
I would remove it in master, for backward compatibility, just in case.

The API of onchange changed in v17. I don't think the python could return command CLEAR anymore (that's why we don't support it in _applyCommands). That command was extensively used to redefine the whole relation (typically, `[[5], [4,1], [4,2]]` stood for "clear everything, then link 1 and 2"). Now onchange only communicates the diff, and with that mindset, we don't really need command 5 anymore. So if there's a record in the relation and you don't want it, simply unlink (3) it.